src/main/java/org/broadinstitute/barclay/argparser/PositionalArgumentDefinition.java

```diff
@@ -110,6 +110,60 @@ public void validateValues(final CommandLineArgumentParser commandLineArgumentPa
         }
     }
 
+    @Override
+    public String getDocString() { return positionalArgumentsAnnotation.doc(); }
+
+    /**
+     * Return a string with the usage statement for this positional argument.
+     * @param argumentColumnWidth width reserved for argument name column display
+     * @param descriptionColumnWidth width reserved for argument description column display
+     * @return the usage string for this argument
+     */
+    public String getArgumentUsage(final int argumentColumnWidth, final int descriptionColumnWidth) {
+
+        final StringBuilder sb = new StringBuilder();
+        sb.append("--").append("POSITIONAL");
+        sb.append(String.format(" <%s>", getUnderlyingFieldClass().getSimpleName()));
+
+        int labelLength = sb.toString().length();
+        int numSpaces = argumentColumnWidth - labelLength;
+        if (labelLength > argumentColumnWidth) {
+            sb.append("\n");
+            numSpaces = argumentColumnWidth;
+        }
+        Utils.printSpaces(sb, numSpaces);
+
+        String description = getArgumentDescription();
+        if (isDeprecated()) {
```

Comment:
Should this little decorator be factored out so it isn't repeated?  Might make it more consistent if we change things in the future.

Reply:
Done - factored out this decorator plus another method to format the description since that was also duplicated on two places.